src/compiler/shame.d.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @fileoverview The junk-drawer of type definitions for files that do not pass type checking yet.
+ * Shame! Shame! Shame! Avoid adding to this.
+ */
+
+declare module '#builtins/amp-layout/build-dom' {
+  import {BuildDom} from '#compiler/types';
+  export const buildDom: BuildDom;
+}
+
+declare module 'extensions/amp-fit-text/0.1/build-dom' {
+  import {BuildDom} from '#compiler/types';
+  export const buildDom: BuildDom;
+}
+
+declare module '#core/dom' {
+  export const copyChildren: any;
```

Comment:
There's only four of these, let's actually provide the types rather than `any` here, they should all be pretty straightforward